src/NUnitFramework/framework/PublicAPI/net35/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,303 @@
+*REMOVED*NUnit.Framework.Constraints.NUnitEqualityComparer.AreEqual(object x, object y, ref NUnit.Framework.Constraints.Tolerance tolerance, bool topLevelComparison = true) -> bool
```

Comment:
This is no good. We will almost certainly break someone. We need to revert this before our next release.

Reply:
This was probably me in #3406
I had to update the signature, and since I've had issues with unexpectedly breaking changes of default method args in the past, I thought it safest at the time to split that function up into two overloads.

Sorry, I hadn't expected it to be considered breaking

Reply:
No, sorry on my end... I wasn't looking at what we replaced it with. I don't think this is either a binary break or a source break; we made the parameter no longer be a default parameter, which is a source break but not a binary break, and then we added an overload to avoid the source break. The subtlety is that you've switched overloads on recompile, but in this case that's exactly what we want anyway.

So we're good!